static/retirement/aggregatedsearch/index.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Retirement of the data endpoint for aggregated search pages
+---
+
+## Background
+
+In line with our [approach to the retirement of API endpoints](../), it has been decided that ONS will retire the data endpoints for our aggregated search pages due to the following reasons:
+
+- out of date technology
+- built as a proof of concept
+- strategic technology direction is elsewhere
+- cost of upkeep is too high with similar services available
+
+The UI for these pages has already been moved to using our Search API and it is the intention of ONS to separate data and UI endpoints by domain in future. Data in JSON format will be accessible via our API found at <https://api.beta.ons.gov.uk/v1/search>. You can also browse the [full API Swagger specification](/search/search).
```

Comment:
Should "using" be "use" here?

Reply:
yeah probably :D

Reply:
https://github.com/ONSdigital/dp-developer-site/pull/92